package.json

```
@@ -65,7 +65,7 @@
         "conventional-changelog-cli": "^2.0.34",
         "flow-bin": "0.155.0",
         "fs-extra": "^9.0.1",
-        "grumbler-scripts": "^3.0.70",
+        "grumbler-scripts": "5.0.0",
```

Should we keep the carrot here so we get patches and minor versions for free?
```suggestion
        "grumbler-scripts": "^5.0.0",
```

---

consider it done